triggers/runner/src/main/resources/logback.xml

```diff
@@ -1,9 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
-        </encoder>
+    <appender name="STDOUT" class="net.logstash.logback.appender.LoggingEventAsyncDisruptorAppender">
```

Comment:
Using the async disruptor ensures logging is highly efficient (think of a million or so logging events per thread per second with linear scaling effect as we add more logging threads!).